site/src/locales/en-US/common.json

```diff
@@ -2,5 +2,23 @@
   "auth": {
     "login": "Log In",
     "signup": "Sign Up"
-  }
+  },
+  "Search": "Search",
+  "Block Hub": "Block Hub",
+  "Documentation": "Documentation",
+  "Specification": "Specification",
+  "Publish a Block": "Publish a Block",
+  "Contact Us": "Contact Us",
+  "page_header_text": "A POWERFUL NEW PROTOCOL FOR DEVELOPERS",
```

Comment:
**Is it possible to selectively apply CSS based on the i18n profile?** Ideally, we would avoid having all-caps in the translated string here (and instead rely on CSS to text-transform to all-uppercase). However, I am wary of applying text-transforms such as this blanketly to all translations for some of the reasons outlined in [this comment](https://github.com/meetuparchive/sassquatch2/issues/144#issuecomment-118958029).

Reply:
Ok, I'll think about it~